common/src/Renderer/BrushRenderer.h

```diff
@@ -216,6 +208,7 @@ namespace TrenchBroom {
             void setTintColor(const Color& tintColor);
             void setShowOccludedEdges(bool showOccludedEdges);
             void setOccludedEdgeColor(const Color& occludedEdgeColor);
+            void setTransparent(bool transparent);
```

Comment:
We should clarify what `setTransparent` does

Reply:
Agreed.